config/locales/en.yml

```diff
@@ -108,6 +108,8 @@ en:
       invalid_project_media_archived_value: Sorry, archived value not supported
       invalid_fact_check_language_value: Sorry, language value not supported
       fact_check_empty_title_and_summary: Sorry, you should fill title or summary
+      invalid_feed_licenses_value: Sorry, licenses value not supported
+      invalid_feed_saved_search_value: Sorry, list should belong to one of feeds team
```

Comment:
Fix: You're calling these strings with `errors.add(:licenses [...])`, so the error message will be:

```
Licenses Sorry, licenses value not supported
```

This happens because it concatenates the first and second arguments passed to `errors.add`. In that case, the messages could be:

```
value not supported
should belong to a workspace that is part of this feed
```

Let's use "workspace" instead of "team" for user-facing messages.

Reply:
Done